LICENSE

```diff
@@ -186,7 +187,7 @@ Apache License
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [2020] [Carsten Ehbrecht]
+   Copyright [yyyy] [name of copyright owner]
```

Comment:
Should this be set to a real year and owner name?

Reply:
No, this is the boilerplate license file that needs to be in the files.

In order to apply the license, the block of text itself needs to be added into the source files, with the changes for year and authorship. Learned this not too long ago.

There are some more issues around using the Apache v2.0 License as well; I might bring this up in our next team meeting.